processor/src/trace/mod.rs

```diff
@@ -241,7 +241,7 @@ fn finalize_trace(process: Process, mut rng: RandomCoin) -> (Vec<Vec<Felt>>, Aux
     assert_eq!(clk, stack.trace_len(), "inconsistent stack trace lengths");
 
     // Add the range checks required by the auxiliary table to the range checker.
-    range.add_lookups(aux_table.get_range_checks());
+    aux_table.update_range_checks(&mut range);
```

Comment:
minor nit: would `append_range_check` be a better name here?

Reply:
Yes, thank you! I couldn't think of one I liked. I'll update it